insights/parsers/up2date_log.py

```diff
@@ -0,0 +1,7 @@
+from .. import LogFileOutput, parser
+
+
+@parser('up2date.log')
+class Up2dateLog(LogFileOutput):
```

Comment:
`Up2DateLog`?

Reply:
the log file name is /var/log/up2date,  as the up2date is one word, so Up2dateLog

Reply:
not change this as my last comment